docs/backup_and_restore.rst

```diff
@@ -334,6 +338,74 @@ Migrating Using a V2+V3 or V3-Only Backup
 #. :doc:`Test the new instance <test_the_installation>` to verify that the
    web interfaces are available and the servers can be reached via SSH.
 
+.. _repair_admin_usbs:
+
+Repair Additional Admin Workstations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have additional *Admin Workstation* USBs, they will no longer have
+valid SSH credentials and will need to be repaired. In these steps, the "primary
+*Admin Workstation*" is the one which you used to complete the above migration 
+process.
+
+#. Prepare a fresh :doc:`Transfer Device <set_up_transfer_and_export_device>`.
+   In this case, use LUKS encryption and set a strong (7-word diceware) 
+   passphrase, which you may record in your primary *Admin Workstation*
+   KeePassXC password manager. 
+
+#. Copy the following files from your primary *Admin Workstation* onto the
+   Transfer Device:
+   
+   - ``~/Persistent/securedrop/install_files/ansible-base/tor_v3_keys.json``
+   - ``~/Persistent/securedrop/install_files/ansible-base/mon-ssh.auth_private``
+   - ``~/.ssh/id_rsa.pub``
+   - ``~/.ssh/id_rsa``
+
+#. Boot into each additional Admin Workstation, setting an administrative 
+   passphrase during startup. Once logged in, attach the Transfer Device
+   and unlock it.
+
+#. Ensure that this Admin Workstation is using an up-to-date version of Tails
+   and is running the latest SecureDrop Application code, |version|.
+   
+#. As you did with the primary *Admin Workstation* archive the existing 
+   SSH configuration:
+
+   .. code:: sh
+
+       find ~/.ssh/ -type f -exec mv {} {}.bak \;
+
+#. From the Transfer Device, copy ``~/.ssh/id_rsa`` and 
+   ``~/.ssh/id_rsa.pub`` to the ``~/.ssh/`` directory. 
+
+#. From the Transfer Device, copy ``tor_v3_keys.json`` and 
+   ``mon-ssh.auth_private`` to the 
+   ``~/Persistent/securedrop/install_files/ansible-base`` directory. 
+
+#. In the Terminal, type the following commands:
+  
+   .. code:: sh
+
+      cd ~/Persistent/securedrop
+      ./securedrop-admin tailsconfig
+
+#. Test connectivity to each server by running ``ssh app uptime`` 
+   and ``ssh mon uptime``.       
+
+#. Once all *Admin Workstations* have been updated, securely wipe the files on 
+   the Transfer Device, by right-clicking them in the file manager and selecting 
+   **Wipe**. Do not skip this step. Then, reformat the Transfer device using the
```

Comment:
I would remove "Do not skip this step". If you want the Wipe/Format steps to stand out clearly, we could separate them.